pkg/webhook/webhook.go

```diff
@@ -26,15 +26,10 @@ import (
 	trustapi "github.com/cert-manager/trust-manager/pkg/apis/trust/v1alpha1"
 )
 
-// Options are options for running the wehook.
-type Options struct {
-	Log logr.Logger
-}
-
 // Register the webhook endpoints against the Manager.
-func Register(mgr manager.Manager, opts Options) error {
-	opts.Log.Info("registering webhook endpoints")
-	validator := &validator{log: opts.Log.WithName("validation")}
+func Register(mgr manager.Manager, log logr.Logger) error {
+	log.Info("registering webhook endpoints")
+	validator := &validator{}
```

Comment:
suggestion: I don't think we need to pass the logger here for this message, it doesn't seem worth it to me. I'd vote to move `log.Info("registering webhook endpoints")` to where the `Register` function is called and drop the logger here entirely

Reply:
Agreed! Will fix.

Reply:
Fixed now.